src/tools/pkcs11-tool.c

```diff
@@ -6271,6 +6269,7 @@ static int test_decrypt(CK_SESSION_HANDLE sess)
 				continue;
```

Comment:
The above message, should not be follow-up of the above line, but it should look like
```
printf("    %s: mechanism can't be used to decrypt, skipping\n", p11_mechanism_to_name(mechs[n]));
```
similarly as the first print inside of the `encrypt_decrypt()`.

The current version says the mechanism does not work for decrypt, but it does not say which one.